WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1398,11 +1416,16 @@ import WordPressComAnalytics
 extension ReaderStreamViewController : ReaderStreamHeaderDelegate {
 
     public func handleFollowActionForHeader(header:ReaderStreamHeader) {
+        guard let topic = readerTopic else {
+            return
+        }
         // Toggle following for the topic
-        if readerTopic!.isKindOfClass(ReaderTagTopic) {
-            toggleFollowingForTag(readerTopic as! ReaderTagTopic)
-        } else if readerTopic!.isKindOfClass(ReaderSiteTopic) {
-            toggleFollowingForSite(readerTopic as! ReaderSiteTopic)
+        if topic.isKindOfClass(ReaderTagTopic) {
+            toggleFollowingForTag(topic as! ReaderTagTopic)
```

Comment:
Just wondering if this is the same as:

`if let topic = readerTopic as? ReaderTagTopic { }`